specs/merge/beacon-chain.md

```diff
@@ -69,15 +67,59 @@ This patch adds transaction execution to the beacon chain as part of the Merge f
 #### `BeaconBlockBody`
 
 ```python
-class BeaconBlockBody(phase0.BeaconBlockBody):
+class BeaconBlockBody(Container):
```

Comment:
Curious, why `class BeaconBlockBody(altair.BeaconBlockBody)` doesn't work here?

Reply:
See `BeaconState` comment below. I did the same with this type for consistency.